distributed/scheduler.py

```diff
@@ -161,14 +163,6 @@ def nogil(func):
 DEFAULT_DATA_SIZE = declare(
     Py_ssize_t, parse_bytes(dask.config.get("distributed.scheduler.default-data-size"))
 )
```

Comment:
could not move this as it is required by TaskState, and it would be way too expensive to reload it on every ``TaskState.__init__``